skinning/judgement.md

```diff
@@ -0,0 +1,13 @@
+# Judgement
+These are the files that determine what image shows up for each of the judgements in-game. The files are labled as such:
+
+|Filename|Judgement|
+|---|---|
+|`flawless.png`|Maximum Hit Judgement|
+|`perfect.png`|300 Hit Judgement|
```

Comment:
These numbers are from osu!. They have nothing to do with this.

Reply:
Should I just copy this game's judgement names?

Reply:
Should be fine yeah